internal/server/server.go

```diff
@@ -49,7 +49,24 @@ func RunServer(config *Config) error {
 	logger := log.NewCtxLogger(config.LogLevel, []string{domain.TraceIDKey})
 	crypto := crypto.NewAES(config.EncryptionSecretKeyKey)
 	validator := validator.New()
-	notifier, err := notifiers.NewClient(&config.Notifier, logger)
+	notifierConfig := notifiers.ConfigMultiClient{}
+	if &config.Notifier != nil {
+		// map old to the new format
+		var provider = config.Notifier.Provider
+		notifierConfig = notifiers.ConfigMultiClient{
+			Notifiers: map[string]notifiers.Notifier{
```

Comment:
here we need to map `config.Notifier.SlackConfig.Workspaces` into `notifiers.Notifier` as well. For a `provider: slack` notifier with multiple workspaces, each workspace config should be treated as a single `provider: slack` notifier